stories/src/ui/ImageCard.stories.mdx

```diff
@@ -0,0 +1,49 @@
+import { Meta, Story, Preview } from '@storybook/addon-docs/blocks'
+import {
+  Heading,
+  ImageCard,
+  ImageCardContent,
+  Paragraph,
+} from '@amsterdam/asc-ui'
+
+<Meta title="UI/ImageCard" component={ImageCard} />
+
+# ImageCard
+
+Doesn't exist in Design System yet
+
+**Component status: Experimental**
+
+## Default
+
+<Preview>
+  <Story name="Default">
+    <ImageCard backgroundImage="http://lorempixel.com/output/food-q-c-640-480-3.jpg">
```

Comment:
This one looks massive. In the old version it looked a lot smaller. Maybe you can wrap it a div with only 50% width or something

Reply:
ja dat is een goed punt: ik ga even de max-widh op alle componenten zetten

Reply:
gedaan